### ISSUE TITLE
Gracefully omit fields

### ISSUE THREAD

**Issue by Scitz0** [jdereg/json-io#93]
My application is extendable with plugins that can be loaded and unloaded. Data from these plugins can be stored on the objects I use json-io to serialize to disk. So its not uncommon that when deserializing the json-io file some field/object in this key map reference an unloaded plugin/class that doesn't exist. In this case, does json-io return null or does it throw an exception? My wish would be that it just ignored the field and continued with the rest.

When I previously used XMLEncoder/XMLDecoder to do the job I used setExceptionListener() and in the custom ExceptionListener checked if the exception was of instance ClassNotFoundException or NullPointerException and in those cases just suppressed them, else I logged the error and continued.

Is this possible in json-io or is it something that could be added?

**Comment by jdereg** [comment]
Can you write a simple unit test that checks this out?  I believe it will ignore fields it cannot set on the target object.  Should be easy enough to test.

**Comment by Scitz0** [comment]
Was away for a week so just now got around to do some testing and you are correct, it does omit fields that it cant recognize without throwing any exception and successfully loads the object. 

When I did my quick test before adding the issue I manually edited the json and got an exception loading it but I now saw that I had a small typo in the json format after editing the file. When corrected it works as expected.
